src/Magnum/ImGuiIntegration/Context.h

```diff
@@ -251,6 +251,35 @@ If you don't do that, the fonts stay at the original scale, not matching the
 new UI scaling anymore. If you didn't supply any custom font, the function will
 reconfigure the builtin font automatically.
 
+@section ImGuiIntegration-Context-large-meshes Large meshes
+
+Complex user interfaces or widgets like [ImPlot](https://github.com/epezent/implot)
+may end up creating large meshes with more than 64,000 vertices. Because ImGui
+defaults to 16-bit index buffers this can lead to asserts or visual errors.
+
+If the underlying GL context supports @ref GL::Mesh::setBaseVertex() "setting the base vertex for indexed meshes"
+the rendering backend sets the @cpp ImGuiBackendFlags_RendererHasVtxOffset @ce
+flag. This lets ImGui know the backend can handle per-draw vertex offsets,
+allowing meshes with a little over 2 million vertices. Support for that
+requires one of the following:
+
+-   OpenGL 3.2 and higher or extension `ARB_draw_elements_base_vertex`
+-   OpenGL ES 3.2 or extension `OES_draw_elements_base_vertex` or
+    `EXT_draw_elements_base_vertex`
+-   WebGL 2.0 and extension `WEBGL_draw_instanced_base_vertex_base_instance`
```

Comment:
There's a few custom Doxygen commands that provide links to extensions and cross-link this page with an index of all features requiring a particular version/extension:

```suggestion
@requires_gl32 Extension @gl_extension{ARB,draw_elements_base_vertex}
@requires_gles32 Extension @gl_extension{OES,draw_elements_base_vertex} or
    @gl_extension{EXT,draw_elements_base_vertex}
@requires_webgl_extension WebGL 2.0 and extension
    @webgl_extension{WEBGL,draw_instanced_base_vertex_base_instance}
```